server/push_test.go

```diff
@@ -240,11 +246,39 @@ func TestShouldUpdateLastUsed(t *testing.T) {
 	assert.True(lu.Before(s.rtmpConnections["mani1"].lastUsed))
 }
 
+func ignoreRoutines() []goleak.Option {
+	funcs2ignore := []string{"github.com/golang/glog.(*loggingT).flushDaemon", "go.opencensus.io/stats/view.(*worker).start",
```

Comment:
Maybe add a comment here describing what this function does and why we need to exclude these particular functions?

Reply:
added